content/en/examples/security/kind-with-namespace-level-baseline-pod-security.sh

```diff
@@ -25,4 +27,4 @@ spec:
       ports:
         - containerPort: 80
 EOF
-kubectl apply -n example -f /tmp/pss/nginx-pod.yaml
+kubectl apply -n example -f -
```

Comment:
Same question about reusing the example pod.

Reply:
See https://github.com/kubernetes/website/pull/37587/files#r1086070175

Relying on the wifi at a conference. Even in 2023, it's for the brave of heart :wink: